### PR TITLE
Pyblake2 package removal

### PR DESCRIPTION
**Pull request by ptagl**
Pyblake features are now integrated into the standard "hashlib" module, so `pyblake2` is not required to be installed/imported anymore.

### CODE REVIEW

**Comment by titusen** [comment]
In test_joinsplit.cpp:257 there are some comments left using pyblake2. I guess it can be cleaned up, please.

File: contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN set -euxo pipefail \
     && apt-get -y --no-install-recommends install aria2 autoconf automake bsdmainutils build-essential ca-certificates \
        cmake curl dirmngr fakeroot git g++-multilib gnupg2 help2man libc6-dev libgomp1 libtool lintian m4 ncurses-dev \
        pigz pkg-config pv python3-dev python3-pip python3-setuptools python3-wheel time unzip wget zlib1g-dev \
-    && pip3 install pycryptodome b2 pyblake2 pyzmq websocket-client pyzmq \
```
Comment (JackPiri) [inline contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile:19]: Just seen by chance, can we get rid of that `pyzmq` being written a second time?

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

**Review by drgora — APPROVED** [review]
Looks good to me

**Review by a-petrini — APPROVED** [review]
Looks good, thanks for his PR!

File: contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile (hunk 1 above)
Comment (ptagl) [reply contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile:19]: Nice catch, removed, thanks!

**Comment by ptagl** [comment]
> In test_joinsplit.cpp:257 there are some comments left using pyblake2. I guess it can be cleaned up, please.

@titusen thanks, I've updated and tested that Python script, it's not critical but anyway useful to regenerate data for the test vectors.

**Review by titusen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**